.github/workflows/examples.yml

```diff
@@ -87,7 +87,7 @@ jobs:
         python examples/cfd/example_diffusion.py
 
     - name: Upload coverage to Codecov
-      if: matrix.name != 'pytest-docker-py37-gcc-omp'
+      if: matrix.name != 'pytest-docker-py38-gcc-omp'
```

Comment:
This isn't right, there is no pytest or matrix name in this workflow it's only run once, if should be removed